Alicerce/Sources/Logging/Providers/ConsoleProvider.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+final class ConsoleProvider {
+    
+    internal var minLevel: Log.Level = .error
+    internal var formatter: LogItemFormatter = LogItemStringFormatter()
+    internal var output: ConsoleOutput = .print
+}
+
+extension ConsoleProvider {
+    public enum ConsoleOutput {
+        case print
+        case nslog
```

Comment:
it was a feature that I noticed in SwiftyBeaver, was easy to implement so I brought to ours. Completely agree with the background threaded approach of writing to the log; will do this as soon as the *basic* improvements are through